test/fixtures/test-hex-id-data.js

```diff
@@ -530,7 +530,13 @@ mergedH3Layer.config = {
   isVisible: true,
   highlightColor: [252, 242, 26, 255],
   isConfigActive: false,
-  colorField: fields[1],
```

Comment:
because we are now loading layers before filters, fields are not updated with filterProps.

Reply:
that's going to be a problem. we need field.mappedValue to calculate layer domain